src/javascript/DesignSystem/Card/Card.jsx

```diff
@@ -23,7 +23,7 @@ const styles = theme => ({
         }
     },
     containerSelected: {
-        border: `1px solid ${theme.palette.brand.alpha}`
+        border: `1px solid ${theme.palette.brand.alpha}!important`
```

Comment:
I think we can do a better and stronger CSS selector to avoid `!important` :)

Reply:
```suggestion
        "article&":  {
              border: `1px solid ${theme.palette.brand.alpha}`
         }
```

Reply:
Okay :D